beginner_source/transfer_learning_tutorial.py

```diff
@@ -339,6 +340,39 @@ def visualize_model(model, num_images=6):
 plt.ioff()
 plt.show()
 
+######################################################################
+# Save and load the model
+# ----------------------
+#
+# Here we have saved the trained model and loaded it for inference. We can 
+# now use our trained model to make predictions on our own images and analyze 
+# the results.
+#
+
+def save_and_load_model(model, model_name):
```

Comment:
Is this a convoluted way to do a deepcopy?
I think in this case using deepcopy is fine if it works.